powerfactory/powerfactory-model/src/test/resources/scheme.json

```diff
@@ -0,0 +1,21 @@
+{
+  "classes" : [ {
+    "name" : "ElmFoo",
+    "attributes" : [ {
+      "name" : "loc_name",
+      "type" : "STRING"
+    }, {
+      "name" : "i",
+      "type" : "INTEGER"
+    } ]
+  }, {
+    "name" : "ElmBar",
+    "attributes" : [ {
+      "name" : "loc_name",
+      "type" : "STRING"
+    }, {
+      "name" : "d",
+      "type" : "DOUBLE"
+    } ]
+  } ]
+}
```

Comment:
> Maybe we could support the .json import/export directly through the Importer interface of PowerFactory ?

This will be done in #2035 but not with the study case json but with the project json.